README.md

```diff
@@ -216,6 +216,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [VueJS Dublin](https://www.vuejsdublin.com/) - Website for the local Vue meetup group in Dublin, Ireland.
 - [Scale Leap](https://www.scaleleap.com/) - Custom Software Development For Amazon Sellers.
 - [El Cable Amarillo](https://www.elcableamarillo.cc/) - Open source project for the use of programming and robotics in educational centers using software tools and free hardware.
+- [Zip Stores](https://zip.co/stores/) - Zip Pay and Zip Money Stores Directory.
```

Comment:
Could you provide a better explanation? :relaxed: 

Also, I can't see Nuxt.js being used on the page.